tests/test_merkleization_types.nim

```diff
@@ -95,7 +95,7 @@ let
     q: [E(x: false, y: true), E(x: true, y: false)],
     r: List[E, 2](@[E(x: false, y: true), E(x: true, y: false)]),
     s: List[E, 2](@[]),
-    t: SingleMemberUnion[E](selector: 0, value: E(x: false, y: true)),
+    t: List[E, 2](@[]),
```

Comment:
could relabel the followups instead of repeating the same test twice, but alright

Reply:
There were other repeats or near-repeats, wasn't sure of the intent there. Having exactly the same number of tests as letters in the alphabet seemed likely intentional, given that.

So, probably for a followup PR either way, but can relabel things, if it wasn't in fact the intent to just fill space with enough tests to go exactly, no more and no less, from literal `a` to literal `z`.